addons/l10n_hu_edi/wizard/account_move_send.py

```diff
@@ -36,19 +36,30 @@ def _get_wizard_values(self):
     def _compute_l10n_hu_edi_enable_nav_30(self):
         for wizard in self:
             enabled_moves = wizard.move_ids.filtered(lambda m: 'upload' in m._l10n_hu_edi_get_valid_actions())._origin
-            if wizard.mode in ('invoice_single', 'invoice_multi') and enabled_moves:
-                wizard.l10n_hu_edi_enable_nav_30 = True
-                wizard.l10n_hu_edi_actionable_errors = enabled_moves._l10n_hu_edi_check_invoices()
-
-            else:
-                wizard.l10n_hu_edi_enable_nav_30 = False
-                wizard.l10n_hu_edi_actionable_errors = False
+            wizard.l10n_hu_edi_enable_nav_30 = wizard.mode in ('invoice_single', 'invoice_multi') and enabled_moves
 
-    @api.depends('l10n_hu_edi_enable_nav_30', 'l10n_hu_edi_actionable_errors')
+    @api.depends('l10n_hu_edi_enable_nav_30')
     def _compute_l10n_hu_edi_checkbox_nav_30(self):
         for wizard in self:
             wizard.l10n_hu_edi_checkbox_nav_30 = wizard.l10n_hu_edi_enable_nav_30
 
+    @api.depends('l10n_hu_edi_enable_nav_30', 'l10n_hu_edi_checkbox_nav_30', 'move_ids')
+    def _compute_l10n_hu_edi_actionable_errors(self):
+        for wizard in self:
+            if wizard.l10n_hu_edi_enable_nav_30:
+                enabled_moves = wizard.move_ids.filtered(lambda m: 'upload' in m._l10n_hu_edi_get_valid_actions())._origin
+                actionable_errors = enabled_moves._l10n_hu_edi_check_invoices()
+
+                if enabled_moves and not wizard.l10n_hu_edi_checkbox_nav_30:
+                    actionable_errors['checkbox_not_ticked'] = {
+                        'message': _("Invoices issued in Hungary must, with few exceptions, be reported to the NAV's Online-Invoice system!")
```

Comment:
Isn't the message a bit too alarmist if there are exceptions ? :thinking:

Reply:
It was approved by CHKL, but if you want, we can ask him for his opinion again when he comes back.